Assets/Scripts/Game/UserInterfaceWindows/DaggerfallPauseOptionsWindow.cs

```diff
@@ -77,6 +81,18 @@ protected override void Setup()
             //loadButton.BackgroundColor = DaggerfallUI.DaggerfallUnityNotImplementedColor;
             loadButton.OnMouseClick += LoadButton_OnMouseClick;
 
+            // Sound Bar
+            Button soundPanel = DaggerfallUI.AddButton(new Rect(6.15f, 23.20f, barMaxLength, 5.5f), optionsPanel);
```

Comment:
You shouldn't need to use part-pixel coordinates here, but I know why you had to. This is due to a scaling bug when Daggerfall-native canvases are scaled and centred in parent screen-native canvas. I'll try to fix this after merging PR.